aws/cloudformation-templates/template.yaml

```diff
@@ -180,6 +189,7 @@ Resources:
         ExperimentStrategyTable: !GetAtt Base.Outputs.ExperimentStrategyTable
         ParameterPersonalizeEventTrackerId: !GetAtt Base.Outputs.ParameterPersonalizeEventTrackerId
         ParameterAmplitudeApiKey: !GetAtt Base.Outputs.ParameterAmplitudeApiKey
+        ParameterOptimizelySdkKey: !GetAtt Base.Outputs.OptimizelySdkKey
```

Comment:
Should be `!GetAtt Base.Outputs.ParameterOptimizelySdkKey` rather than `!GetAtt Base.Outputs.OptimizelySdkKey`